src/components/TodoList.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div>
+   <v-btn @click="onShowForm" color="blue">Add Todo</v-btn>
+   <v-data-table :headers="headers" :items="todos" class="elevation-1">
+      <template v-slot:body="{ items }">
+        <tr v-for="(item, index) in items" :key="index">
+          <td>{{ item.task }}</td>
+          <td>{{ item.description }}</td>
+          <td>{{ item.assignedTo }}</td>
+          <td>
+            <v-btn color="blue" small @click="editTask(item, index)">
+              Edit
+            </v-btn>
+          </td>
+          <td>
+            <v-spacer></v-spacer>
+            <!-- button -->
+            <v-dialog
+             v-model="dialog"
+             persistent
+              max-width="290">
+              <template v-slot:activator="{ on, attrs }">
+              <v-btn color="red" small
+              v-bind="attrs"
+              v-on="on">
+              DELETE
+              </v-btn>
+              </template>
+              <v-card>
+              <v-card-title class="text-h5">
+                DO you want to detele this ?
+              </v-card-title>
+              <v-card-text  text-color="orange">think again before u delete this.Once done ,you cannot retrieve it.
+              </v-card-text>
+              <v-card-actions>
+              <v-spacer></v-spacer>
+              <v-btn
+              color="green"
+              @click="dialog=false">
+              CLOSE
+              </v-btn>
+              <v-btn
+              color="red"
+              @click="dialog = deleteTask (item)">
```

Comment:
why here dialog = deleteTask.? since we are calling a method we just need @click="deleteTask". And if we are assigning any value directly here means we can assign like this, just like the close button of form.